packages/styletron-react/src/index.js

```diff
@@ -85,6 +86,43 @@ class DevProvider extends React.Component<
 export const Provider =
   __BROWSER__ && __DEV__ ? DevProvider : StyletronContext.Provider;
 
+// DEVTOOLS SETUP
+type StyletronStyles = {
+  classes?: any,
+  styles?: any,
+  extends?: any,
+};
+if (__BROWSER__ && __DEV__ && !window.__STYLETRON_DEVTOOLS__) {
+  const atomicMap = new Object();
+  const extensionsMap = new WeakMap();
+  const stylesMap = new WeakMap();
+  const getStyles: StyletronStyles = element => {
+    const styles: StyletronStyles = {};
+    if (stylesMap.has(element)) {
+      styles.styles = stylesMap.get(element);
+      if (element.classList.length) {
+        const classes = {};
+        for (const className of element.classList) {
+          // $FlowFixMe
```

Comment:
What's the reason for this `$FlowFixMe`? In any case, I think coercion is preferable if it's something not going to be changed ever.